chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
```

Comment:
Exactly.  A make rule such as `foo: bar` with no recipe is a NOOP with no function other than to just confuse.